package.json

```diff
@@ -17,10 +20,34 @@
     "pretest": "npm run compile",
     "posttest": "npm run lint",
     "slug": "ts-node slug.ts",
-    "test": "jest"
+    "test": "jest",
+    "cow-slug": "cow-slug.ts",
```

Comment:
Will this work? Shouldn't be the same as "start"?

Reply:
yeah I tested this, it works with npm run test but maybe not best practice then?
<img width="791" alt="image" src="https://user-images.githubusercontent.com/108457192/188447276-a043dfa6-4ed2-4504-ad2e-a3cab2c482f8.png">
I don't no how it would work with start, the jest keyword just search for a test file I guess?

Reply:
The assessment requires you to do a "npm run cow-slug". The test isn't the program running itself :)

Reply:
aha, like this ?
<img width="562" alt="image" src="https://user-images.githubusercontent.com/108457192/188460396-46c7485c-b821-4f72-87a4-33b6644eed14.png">